### PR TITLE
coolprop-refprop-headers: submission of new port

### PR DESCRIPTION
**Pull request by jsalort**
#### Description

Submission of coolprop-refprop-headers port.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.6.1 21G217 arm64
Xcode 14.2 14C18

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint --nitpick`?
- [X] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: science/coolprop-refprop-headers/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+checksums           rmd160  248da34d6e3f6b6e7140c9d2768c9d6618e74119 \
+                    sha256  da3cbbb49f24c0b03f66594d3e8735cb13b779ab0e4652bd918570cf81cf67d7 \
+                    size    15299
+
+maintainers         {juliensalort.org:mp @jsalort} openmaintainer
+categories-append   science
+
+description         Header files for the REFPROP fluid property library.
+long_description    Header files for the REFPROP fluid property library.
```
Comment (reneeotten) [inline science/coolprop-refprop-headers/Portfile:20]: just use "{*}${description}"

**Review by reneeotten — CHANGES_REQUESTED** [review]
should this also set "install_libs no"?

**Comment by jsalort** [comment]
> should this also set "install_libs no"?

I don't know this setting. A search through macports-ports Portfile found no example.
```tclsh
install_libs no
```
yields `Failed to parse file science/coolprop-refprop-headers/Portfile: invalid command name "install_libs"`.

Do you mean
```tclsh
set install_libs        no
```
Is it actually useful ?

**Comment by kencu** [comment]
installs_libs no

https://github.com/macports/macports-ports/blob/3e42635a01c4016d41c3be8da721e11f640d4f6f/devel/cmake/Portfile#L25

**Comment by jsalort** [comment]
> installs_libs no
> 
> https://github.com/macports/macports-ports/blob/3e42635a01c4016d41c3be8da721e11f640d4f6f/devel/cmake/Portfile#L25

Ah! Got it. Thank you.

**Pull request merged**